src/app/routes/certRegRoutes.js

```diff
@@ -0,0 +1,31 @@
+const proxyUtils = require('../proxy/proxyUtils.js')
+const permissionsHelper = require('../helpers/permissionsHelper.js')
+const envHelper = require('../helpers/environmentVariablesHelper.js')
+const certRegURL = envHelper.LEARNER_URL
+const reqDataLimitOfContentUpload = '50mb'
+const proxy = require('express-http-proxy')
+const logger = require('sb_logger_util_v2')
+
+module.exports = function (app) {
+
+    app.all('/certreg/v1/certs/search',
```

Comment:
Define proxy object on top & use that object.

Reply:
done